docs/build_parameters.rst

```diff
@@ -254,10 +252,37 @@ to the root of the object::
                         "required": ["openidc_dir"]
                     }
                 ]
+            },
+            "signing_intents": {
+                "description": "List of signing intents",
+                "type": "array",
+                "items": {
+                    "properties": {
+                      "name": {
+                        "description": "signing intent name",
+                        "type": "string"
+                      },
+
+                      "keys": {
+                          "description": "signing intent keys",
+                          "type": "array",
+                          "items": {
+                              "type": "string"
```

Comment:
Could this be more restrictive? Like hexadecimal?

Reply:
`"pattern": "^[a-fA-F0-9]+$"` ?